Assets/Scripts/Combat/Combantants/EnemyCombatant.cs

```diff
@@ -67,11 +70,13 @@ protected override void TakeTurnWhileDead()
     protected override void TakeTurnWhileAlive()
     {
         DisplayUI();
-        enemyUI.GetComponent<Text>().text = "I Attak! But actually i slep";
+        EnemyUI.GetComponent<Text>().text = "I Attak!";
 
-        StartCoroutine(SimpleEnemyAttack());
+        Debug.Log(Abilities.Length);
+        Abilities[Random.Range(0, Abilities.Length - 1)].StartAbility(false);
```

Comment:
hmm we could eventually update that to have an attack pattern instead if we want to 🤔

Reply:
Eventually is a hard word to stomach when we have 1 week left, but that's a good idea! 😆